test/stepDefinitions/create-caseSteps.js

```diff
@@ -174,6 +174,18 @@ defineSupportCode(function ({ Given, When, Then, And}) {
     await baseSteps.navigateToCreateCasePage();
   });
 
+  When(/^I navigate to multiple pages case type form pages$/, async function (){
+    await baseSteps.navigateToCreateCaseMultiPage();
+    await new CreateCaseWizardPage().clickContinueButton();
+  })
+
+  When(/^I create a case with multiple pages$/, async function (){
```

Comment:
see if we can use the existing step `And I create the case` instead of this, should work then delete this step